app/models/course.rb

```diff
@@ -122,6 +122,8 @@ class Course < ApplicationRecord
 
   has_many :tags, dependent: :destroy
 
+  has_many :course_wikis
```

Comment:
A course should also have many wikis through courses_wikis. That will simplify the controller as well; you should find the course and then do `course.wikis`.